ee/server/service/mdm.go

```diff
@@ -31,6 +32,8 @@ import (
 	"github.com/google/uuid"
 )
 
+var GetABMDeprecatedError = errors.New("This API endpoint has been deprecated. Please use the new GET /abm_tokens API endpoint documented here: https://fleetdm.com/learn-more-about/apple-business-manager-tokens-api")
```

Comment:
Nit, but the convention in Go is to call error _values_ `ErrBlah` and error _types_ `BlahError` (e.g. https://pkg.go.dev/io#pkg-variables - `EOF` being the special exception). Maybe `ErrGetABMDeprecated`?

Or maybe even just returning errors.New inline, as it's only used in this endpoint and may not be worth having an exported error var?

Reply:
Yeah I'm ok with inline! I added this to make it a bit easier to check in integration tests, but since it's just used once I can just do some string checking